### PR TITLE
#53 - Add speaker site

### LINKED ISSUE #53: Add speaker site
Site should:
- include all basic information: name, image, links to social medias, 
- list of presentation with links to youtube, slideshare.

### LINKED ISSUE #54: Add slug for speaker and company
Slug should :
- be generated from names, 
- replace polish characters,
- predict option if two speakers have the same name and surname.
- be added to file: public/api/people.json for every speaker object
- and in public/api/companies.json for every company object

### PR DESCRIPTION
**Pull request by paulinaszkudlarek**
This should close #53 
This should close #54

image
image
image

### CODE REVIEW

**Review by EwelinaSkrzypacz — COMMENTED** [review]
1. If there is no link, I think I will remove hover effect here:

https://user-images.githubusercontent.com/56546832/223112479-7d35397e-2a1d-4337-b0b6-95b5d6030c7f.mp4

Because hover suggest that this link will work. Maybe we should make these buttons look more like they are disabled? Or maybe we should hide icons?

2. When I visit /meetups page, I see a lot of warnings in the dev console:

image

3. This is a side note - on meetup show page - if meetup is in the past I think that text "Zapisz się na to wydarzenie" should be hidden:

image

**Comment by paulinaszkudlarek** [comment]
@EwelinaLasowy I have resolved all issues you mentioned

**Review by EwelinaSkrzypacz — COMMENTED** [review]
1. I think I will hide icons, when there is no link to presentation.
2. When I visit `meetups` page I see some warning in dev console:

image
3.  I don't see anywhere "Sprawdź wydarzenie na Facebooku"
4. When I enter meetup no 22 I see this:

image

and dev console says:

image

**Comment by paulinaszkudlarek** [comment]
@EwelinaLasowy  
1. Icons, are now hidden if links are empty.
2. Resolved
3. Resolved
4. Resolved

File: resources/companies.json [hunk 1]
```diff
@@ -161,4 +161,4 @@
       "lng": 16.925787449461488
     }
   }
-}
+}
```
Comment (EwelinaSkrzypacz) [inline resources/companies.json]:
```suggestion
}

```

File: frontend/style.css [hunk 2]
```diff
@@ -1,3 +1,7 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+:root {
+  --swiper-navigation-color: #3f3f46;
+}
```
Comment (EwelinaSkrzypacz) [inline frontend/style.css]:
```suggestion
}

```

**Review by EwelinaSkrzypacz — DISMISSED** [review]

**Review by MichalMyskow — DISMISSED** [review]

**Review by EwelinaSkrzypacz — COMMENTED** [review]
Why I see empty list "Meetupowy wkład" when counter says that the company has some 
contribution?

image

image

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Review by MichalMyskow — APPROVED** [review]

**Pull request merged**